src/Stryker.CLI/README.md

```diff
@@ -0,0 +1,49 @@
+# Stryker.NET Command Line Interface
+The Stryker CLI is currently the only implemented way to use Stryker.NET. 
+
+## Getting started
+To install Stryker.NET on your *test project* add the following lines to your `.csproj` file.
+
+``` XML
+<ItemGroup>
+    <DotNetCliToolReference Include="dotnet-stryker" Version="0.1.*" />
+    <PackageReference Include="dotnet-stryker" Version="0.1.*" />
+</ItemGroup>
+```
+
+After adding the references, install the packages by executing `dotnet restore` inside the project folder.
+
+## Usage
+To kick off stryker, execute the following command:
+
+`dotnet stryker`
```

Comment:
Where should you execute this command? In the root of your repo? In the test project folder?